[engine] Do not allow to overwrite default composite paths (#3436)
vividus-framework/vividus: docs/modules/ROOT/pages/tests-configuration.adoc

|`report.text-length-diff-threshold`
|`100`
|Defines text threshold length to enable unified diff report publishing on failure, for variable comparisons. If you want to have this report despite the size of the compared data set 0.

> |`engine.composite-paths`

Reviewer: isn't engine too technical for end users?

Please see <<_externalized_configuration>> to get more information about how encryptor password can be passed to the tests.

== Composite steps

> https://jbehave.org/reference/stable/composite-steps.html[Composite steps] allows to reuse set of actions via step with a custom wording.

Reviewer 2: ```suggestion
https://jbehave.org/reference/stable/composite-steps.html[Composite steps] allow to reuse set of actions via custom steps with own wordings.
```

=== How to use composite steps

> . Define composite step:

Reviewer 2: ```suggestion
. Define composite step in `*.steps` file, e.g. `steps/ui/my-app.steps`:
```

+
.Compsite step definition
[source,gherkin]
----
Composite: When I login with username `$username` and password `$password`

> When I enter `<username>` in field located `id(username)`
> When I enter `<password>` in field located `id(password)`
> When I click on element located `id(signInButton)`

Reviewer 2: ```suggestion
When I enter `<username>` in field located by `id(username)`
When I enter `<password>` in field located by `id(password)`
When I click on element located by `id(signInButton)`
```

----

> . Configure composite paths

Reviewer 2: ```suggestion
. Configure paths to files with custom steps
```
